src/System.Runtime.Extensions/src/System/Net/WebUtility.cs

```diff
@@ -45,20 +47,56 @@ public static string HtmlEncode(string value)
                 return value;
             }
 
-            StringBuilder sb = StringBuilderCache.Acquire(value.Length);
```

Comment:
Is the project still using StringBuilderCache after this change?  If not, it can be removed from the .csproj.  If yes, should those places be changed to ValueStringBuilder as well?

Reply:
It's used elsewhere ([eg](https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/src/System/ApplicationId.cs#L44)). I'd be happy to fix those other usages too, in a separate PR, if you like.

Reply:
Hi! Other usages have been removed in #30072